code_folder/22_data_engineering_pipeline.py

```
@@ -0,0 +1,295 @@
+def main():
+    input_dir = get_input_dir()
+    output_dir = get_output_dir()
+    target_col = "Survived"
+
+    # Pipeline stages
+    df = load_extracted_data(input_dir)
+    df = handle_missing_values(df)
+    df = type_cast_and_cleanup(df)
+    df = engineer_features(df)
+    df, encoders = encode_categoricals(df)
+    df, scaler = scale_features(df, target_col)
```

Ruff (0.14.14)

[warning] Unpacked variable `scaler` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)